Examples/restcomm-olympus-nopod/restcomm-olympus/SigninTableViewController.m

```diff
@@ -35,11 +36,17 @@ - (void) viewWillAppear:(BOOL)animated
     [super viewWillAppear:animated];
     if (![Utils isFirstTime]) {
         // Open message view if not already opened
+        // initialize RCDEvice
+        RCDevice *rcDevice = [self registerRCDevice];
```

Comment:
registerRCDevice is scattered in many places (i.e. AppDelegate and SigninTableViewController) and the whole model seems difficult to use. We need to discuss more on possible ways to simplify. Let discuss more about this in a session

Reply:
I agree, Its changed in the branch for the CallKit. The instance is created only in SigninTableViewController